deploy/002-single-token-converter.ts

```diff
@@ -0,0 +1,140 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers, network } from "hardhat";
+import { DeployResult } from "hardhat-deploy/dist/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+type CommonAddresses = {
+  acm: string;
+  oracle: string;
+  primeLiquidityProvider: string;
+  xvsVaultTreasury: string;
+  baseAsset: string;
+  proxyOwnerAddress: string; //Normal timelock
+};
+type Addresses = {
+  bsctestnet: CommonAddresses;
+  bscmainnet: CommonAddresses;
+  sepolia: CommonAddresses;
+  ethereum: CommonAddresses;
+};
+
+const BaseAssets = {
+  bsctestnet: {
+    USDTPrimeConverter: "0xA11c8D9DC9b66E209Ef60F0C8D969D3CD988782c", // USDT
+    USDCPrimeConverter: "0x16227D60f7a0e586C66B005219dfc887D13C9531", // USDC
+    BTCBPrimeConverter: "0xA808e341e8e723DC6BA0Bb5204Bafc2330d7B8e4", // BTCB
+    ETHPrimeConverter: "0x98f7A83361F7Ac8765CcEBAB1425da6b341958a7", // ETH
+    XVSVaultConverter: "0xB9e0E753630434d7863528cc73CB7AC638a7c8ff", // XVS
+  },
+  bscmainnet: {
+    USDTPrimeConverter: "0x55d398326f99059fF775485246999027B3197955", // USDT
+    USDCPrimeConverter: "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d", // USDC
+    BTCBPrimeConverter: "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c", // BTCB
+    ETHPrimeConverter: "0x2170Ed0880ac9A755fd29B2688956BD959F933F8", // ETH
+    XVSVaultConverter: "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63", // XVS
+  },
+  sepolia: {
+    USDTPrimeConverter: "", // USDT
+    USDCPrimeConverter: "", // USDC
+    BTCBPrimeConverter: "", // BTCB
+    ETHPrimeConverter: "", // ETH
+    XVSVaultConverter: "", // XVS
+  },
+  ethereum: {
+    USDTPrimeConverter: "", // USDT
+    USDCPrimeConverter: "", // USDC
+    BTCBPrimeConverter: "", // BTCB
+    ETHPrimeConverter: "", // ETH
+    XVSVaultConverter: "", // XVS
+  },
+};
+
+const addresses: Addresses = {
+  bsctestnet: {
+    acm: "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA",
+    oracle: "0x3cD69251D04A28d887Ac14cbe2E14c52F3D57823",
+    primeLiquidityProvider: "0xAdeddc73eAFCbed174e6C400165b111b0cb80B7E",
+    xvsVaultTreasury: "",
+    baseAsset: BaseAssets.bsctestnet.USDTPrimeConverter,
```

Comment:
This will depend on the SingleTokenConverter we are deploying, it shouldn't be hardcoded to the base asset for the `USDTPrimeConverter` and I don't think we should assume we'll manually change it every time we deploy a SingleTokenConverter

Reply:
Done https://github.com/VenusProtocol/protocol-reserve/pull/48/commits/8ef869acea87739304b75a35b79cacc152133db7